public-api/src/main/java/org/pmiops/workbench/publicapi/DataBrowserController.java

```diff
@@ -1151,9 +1151,9 @@ else if(Integer.parseInt(ard.getStratum3()) == OTHER) {
             }
         } else {
 
-            TreeSet<AchillesResult> maleResults = new TreeSet<AchillesResult>();
-            TreeSet<AchillesResult> femaleResults = new TreeSet<AchillesResult>();
-            TreeSet<AchillesResult> otherResults = new TreeSet<AchillesResult>();
+            ArrayList<AchillesResult> maleResults = new ArrayList<AchillesResult>();
```

Comment:
In Java8 you can just exclude the generic type on the right-hand side `new ArrayList<>()`